src/RA_Integration.vcxproj

```diff
@@ -235,6 +235,7 @@
       <DebugInformationFormat>EditAndContinue</DebugInformationFormat>
       <ControlFlowGuard>false</ControlFlowGuard>
       <ShowIncludes>false</ShowIncludes>
+      <ConformanceMode>true</ConformanceMode>
```

Comment:
> Therefore,  the /permissive- switch had to be turned back off or the code would not compile.

This seems to contradict the comment in the description. I see `/permissive-` in the property pages, and it seems to compile.

Reply:
I should have edited the comment. After I merged the updated master I turned it back on, it was in the commit message but should've been changed in the comment as well.